File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest16332.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;
import org.springframework.web.multipart.MultipartFile;

@RestController
public class BenchmarkTest16332 {

    @PostMapping(path="/BenchmarkTest16332", consumes="multipart/form-data")
    public void BenchmarkTest16332(@RequestPart("file") MultipartFile file, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String uploadName = file != null ? file.getOriginalFilename() : "";
        java.util.function.Function<String,String> transform = v -> v.strip().replaceAll("\\s+", " ");
        String data = transform.apply(uploadName);
        String processed = org.owasp.encoder.Encode.forHtml(data);
        response.getWriter().print("<div>" + processed + "</div>");
    }
}

File: Benchmarks/enterprise/python/django/testcode/benchmark_test_16332.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse
import os


def BenchmarkTest16332(request):
    host_value = request.META.get('HTTP_HOST', '')
    data = '{}'.format(host_value)
    os.system('echo ' + str(data))
    return JsonResponse({"saved": True})
